plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/CustomServerEvaluationStrategy.java

```diff
@@ -99,10 +162,14 @@ public CustomServerEvaluationStrategy(@Nullable @Named("che.docker.ip") String c
 
         // get current ports
         Map<String, List<PortBinding>> ports = containerInfo.getNetworkSettings().getPorts();
-
+        
```

Comment:
Aren't you requesting the exact opposite change as the one you requested [previously](https://github.com/eclipse/che/pull/5366#discussion_r121704027)